core/core-funcs.el

```diff
@@ -253,5 +253,11 @@ result, incrementing passed-tests and total-tests."
      (concat "Hidden Mode Line Mode enabled.  "
              "Use M-x hidden-mode-line-mode to make the mode-line appear."))))
 
+(defun spacemacs/recompile-elpa ()
+  "Recompile packages in elpa directory. Useful if you switch
+Emacs versions."
+  (interactive)
+  (byte-recompile-directory package-user-dir nil t))
```

Comment:
I'm a big fan of self-documenting code, so using `'force` instead of `t` would be better as I wouldn't had to look at documentation to know what is the third argument :wink: